crud/common/utils.lua

```diff
@@ -212,8 +212,8 @@ local function add_nullable_fields_recursive(operations, operations_map, space_f
     return operations
 end
 
--- Tarantool < 2.3 has no fields `box.error.NO_SUCH_FIELD_NO` and `box.error.NO_SUCH_FIELD_NAME`.
-if _TARANTOOL >= "2.3" then
+-- Tarantool < 2.1 has no fields `box.error.NO_SUCH_FIELD_NO` and `box.error.NO_SUCH_FIELD_NAME`.
+if _TARANTOOL >= "2.1" then
```

Comment:
Could you please explain this change?

JSON updates by path was introduced in 2.3 (https://github.com/tarantool/tarantool/commit/9ee5cf82ae579d62543a4bb335dd1e99b00dacf6). I don't understand why do you change it to 2.1.

Reply:
In Tarantool version >= 2.1 there is no `box.error.NO_SUCH_FIELD` error code, there are `box.error.NO_SUCH_FIELD_NO` and `box.error.NO_SUCH_FIELD_NAME` codes (https://github.com/tarantool/tarantool/blob/2.1/src/box/errcode.h#L92 , https://github.com/tarantool/tarantool/blob/2.1/src/box/errcode.h#L208). That's why `test_intermediate_nullable_fields_update` failed for Tarantool version < 2.3.

Reply:
Well, seems all this time we had crud broken at 2.1.

I suggest to move this change into separate patch and corresponding update of targets in CI as well.